### PR TITLE
build(deps): Update altive_lints to 1.11.1

### PR DESCRIPTION
**Pull request by riscait**
## 🔗 Related Issues

- None

## 🙌 What's Done

- 

## ✍️ What's Not Done

- #461

## 🖼️ Image Differences

| Before    | After     |
| --------- | --------- |
| __image__ | __image__ |

## 🤼 Desired Review Method

- [ ] Correction Commit
- [ ] Pair programming

> [!NOTE]
> It is possible that a reviewer's will may cause a method to be implemented that is not selected.

## Pre-launch Checklist
- [x] I have reviewed my own code.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I updated/added relevant documentation (doc comments with ///).

### CODE REVIEW

File: .github/workflows/flutter-app-code-check.yml [hunk 1]
```diff
@@ -60,7 +60,10 @@ jobs:
         uses: bluefireteam/melos-action@v3
 
       - name: Analyze packages
-        run: melos analyze
+        run: melos analyze --fatal-infos
+
+      - name: Run custom lint
+        run: melos run custom_lint
```
Comment (riscait) [inline .github/workflows/flutter-app-code-check.yml:66]: - Add custom lint execution
- To fail even at info level

File: packages/flutter_app/lib/pages/home_page/home_page.dart [hunk 2]
```diff
@@ -22,7 +22,7 @@ class HomePage extends StatelessWidget {
                 t.homePage.appBar.title,
                 style: Theme.of(context).appBarTheme.titleTextStyle?.copyWith(
                   color: context.colorScheme.onInverseSurface,
-                  // Added shadows to improve visibility of text placed over images.
+                  // To improve visibility of text placed over images.
```
Comment (riscait) [inline packages/flutter_app/lib/pages/home_page/home_page.dart:25]: Correct to 80-character limit.

File: packages/util/analysis_options.yaml [hunk 3]
```diff
@@ -1,5 +1 @@
 include: package:altive_lints/altive_lints.yaml
-analyzer:
-  exclude:
-    - "**/gen/*.dart"
-    - "lib/environment/src/firebase_options*.dart"
```
Comment (riscait) [inline packages/util/analysis_options.yaml:5]: Remove unnecessary descriptions as in other packages

**Review by boywithdv — APPROVED** [review]
LGTM


Thank you for updating the altive_lints！🚀

**Pull request merged**